sdk/core/core-amqp/src/requestResponseLink.ts

```diff
@@ -236,7 +235,7 @@ type NormalizedInfo = {
 };
 
 // Handle different variations of property names in responses emitted by EventHubs and ServiceBus.
-const getCodeDescriptionAndError = (props: any): NormalizedInfo => {
+export const getCodeDescriptionAndError = (props: any): NormalizedInfo => {
```

Comment:
Where do we use this outside of core-amqp?

Reply:
Only core-amqp, `src` and `test` though.